### PR TITLE
Winter event

### PR DESCRIPTION
**Pull request by XxSARGExX**
Featurelist:

General:
- Adding Winter Warden Spawn zones in Tau-31 (Ice Asteroids)
- Adding Winter Warden Base and HQ in Tau-31 (Behind harris) as well asl Tau-37 (Between the suns)
- Adding New Winter Warden Ice Asteroids (Mineable Snowballs)
- Adding new Winter Warden Encounter
- Adding new Bomber AI for Winter Warden
- Adding new Transport and Train loadouts
- Adding new Train wreckage in Tau-31
- Adding new Effects

Items:
- Adding Christmas Present as new lootbox
- Adding Naughty Present as new lootbox
- Adding Snowballs as crafting material (Special)
- Adding new SANTA VON CLAUSSEN Equipment 
- Adding Winter Warden Equipment

Changes:
- Adding Snowballs to some Ice Asteroid Drops

Fuse:
- Adding new Small Station Explosion Fuse

### CODE REVIEW

File: Freelancer/DATA/SOLAR/ASTEROIDS/bw10_ice_crystal_field.ini [hunk 1]
```diff
@@ -64,6 +64,14 @@ exclusion = Zone_Bw10_path_img1_2
 exclude_billboards = 1
 exclusion = Zone_Bw10_path_img1_3
 exclude_billboards = 1
+exclusion = ZONE_bw10_Exclusion_wardens_Jumphole
+exclude_billboards = 1
+exclusion = ZONE_Ew01_Exclusion_ice_bw10_Wardens01
```
Comment (erikroe) [inline Freelancer/DATA/SOLAR/ASTEROIDS/bw10_ice_crystal_field.ini:69]: ew01 or bw10?

File: Freelancer/DATA/SOLAR/ASTEROIDS/ew01_wardens_mineable_asteroids.ini [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+placement_radius = 150
+placement_offset = 90
+max_velocity = 10
+max_angular_velocity = 2
+color_shift = 1, 1, 1
+
+[LootableZone]
+zone = Zone_Ew01_Wardens_area
+asteroid_loot_container = lootcrate_ast_loot_niobium
+asteroid_loot_commodity = commodity_niobium
```
Comment (erikroe) [inline Freelancer/DATA/SOLAR/ASTEROIDS/ew01_wardens_mineable_asteroids.ini:51]: Would it be nice to have snowballs be actual loot items here?

File: Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini [hunk 3]
```diff
@@ -3076,3 +3090,300 @@ parent = Bw10_Sun
...
+archetype = ast_icecrystal_largeB
+
+[Object]
+nickname = bw10_ast_icecrystal_largeB_wardens02
+pos = -32220, 0, -38350.0
+rotate = -80, 80, 270
+archetype = ast_icecrystal_largeB
+
+[zone]
+nickname = ZONE_Ew01_Exclusion_ice_bw10_Wardens01
```
Comment (erikroe) [inline Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini:3160]: ew01? But this is BW10

File: Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini [hunk 4]
```diff
@@ -3076,3 +3090,300 @@ parent = Bw10_Sun
...
+faction = gd_ww_grp, 1
+
+[Object]
+nickname = Bw10_surprise_christmas_cv_train01
+ids_name = 460171
+;res str
+; Krampus Carriage
+ids_info = 461643
+;res html
+; The abandoned carriage of Krampus, it got stranded while he was on his way to punish the naughty ones. you found a recipe in the cockpit.\n
```
Comment (erikroe) [inline Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini:3378]: You find a recipe in the cockpit.

File: Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini [hunk 5]
```diff
@@ -3076,3 +3090,300 @@ parent = Bw10_Sun
...
+; \
+; Command: /craft Naughty Present <amount>
+; \
+; \bIngredients\B
+; • 200 Snowballs
```
Comment (erikroe) [inline Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini:3383]: The x is missing here

**Review by erikroe — CHANGES_REQUESTED** [review]

File: Freelancer/DATA/EQUIPMENT/commodity_equip.ini [hunk 6]
```diff
@@ -3276,6 +3401,35 @@ mass = 0
...
+;res html
+; \m\bSnowball\B
+; \l
+; \cYellow\i\bHappy Hollidays!\B\I\C
+; \
+; Snowballs are required for crafting Santa von Claussen's Equipment out of Christmas Present Box as well as the required keys.
+; \
+; \bCrafting Command for the Christmas Key\B 
+; \
+; /craft Christmas Key <amount>
```
[the remark this answers is not in the document]
Comment (XxSARGExX) [reply Freelancer/DATA/EQUIPMENT/commodity_equip.ini:3419]: Thats not true, your crafting plugin doesnt look on the IDS Name it looks on the configuration header name in the brackets [XYZ]

[the remark this answers is not in the document]
Comment (erikroe) [reply Freelancer/DATA/EQUIPMENT/commodity_equip.ini:3419]: Right, sorry, I confused it with the loot boxes

File: Freelancer/DATA/EQUIPMENT/engine_equip.ini [hunk 7]
```diff
@@ -7672,3 +7898,217 @@ outside_cone_attenuation = -3
...
+;; Winter Event
+
+[Engine]
+nickname = santa_engine_fighter_heavy01
+ids_name = 459027
+;res str
+; SANTA’S ENGINE ($class)
+ids_info = 460649
+;res html
+; \m\bSANTA VON CLAUSSEN’S ENGINE Engine\B
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/engine_equip.ini:7911]: It says "ENGINE engine". These duplications should be removed. Same for the following engine infocards

File: Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini [hunk 8]
```diff
@@ -3076,3 +3090,300 @@ parent = Bw10_Sun
...
+faction = gd_ww_grp, 1
+
+[Object]
+nickname = Bw10_surprise_christmas_cv_train01
+ids_name = 460171
+;res str
+; Krampus Carriage
+ids_info = 461643
+;res html
+; The abandoned carriage of Krampus, it got stranded while he was on his way to punish the naughty ones. You find a recipe in the cockpit.\n
```
Comment (erikroe) [inline Freelancer/DATA/UNIVERSE/SYSTEMS/BW10/bw10.ini:3378]: Little correction: You’ll find

**Review by erikroe — CHANGES_REQUESTED** [review]

File: Freelancer/DATA/EQUIPMENT/engine_equip.ini [hunk 9]
```diff
@@ -6799,6 +6799,232 @@ inside_sound_cone = 60
...
+; \l
+; Type: $classLong
+; Speed: -$reverseSpeed to $speed m/s
+; Cruise Speed: $cruiseSpeed m/s
+; Cruise Charge Time: $cruiseChargeTime s
+; Power Draw: $powerUsage /s
+; Cruise Power Draw: $cruisePowerUsage /s
+; Mass: $mass
+; \
+; This modified civilian engines is commonly used by more experienced pilots of the Winter Wardens, it provides high efficency and speed for commercial and tactical usage.
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/engine_equip.ini:6873]: This needs the same corrections as the infocard above

File: Freelancer/DATA/EQUIPMENT/engine_equip.ini [hunk 10]
```diff
@@ -6799,6 +6799,232 @@ inside_sound_cone = 60
...
+outside_cone_attenuation = -3
+inside_sound_cone = 90
+outside_sound_cone = 270
+lootable = true
+
+[Engine]
+nickname = gd_ww_engine_transport01
+ids_name = 459013
+;res str
+; Winter Warden Transport Engine ($class)
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/engine_equip.ini:6912]: Better to shorten it to "Wardens Transport Engine"

File: Freelancer/DATA/EQUIPMENT/engine_equip.ini [hunk 11]
```diff
@@ -6799,6 +6799,232 @@ inside_sound_cone = 60
...
+ids_info = 460635
+;res html
+; \m\bWinter Warden Engine\B
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/engine_equip.ini:6915]: Winter Wardens Transport Engine

File: Freelancer/DATA/EQUIPMENT/engine_equip.ini [hunk 12]
```diff
@@ -6799,6 +6799,232 @@ inside_sound_cone = 60
...
+max_force = 560000
+linear_drag = 7000
+cruise_speed = 500
+lootable = false
+
+[Engine]
+nickname = gd_ww_engine_train01
+ids_name = 459013
+;res str
+; Winter Warden Train Engine ($class)
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/engine_equip.ini:6972]: Same as with the Transport engine

File: Freelancer/DATA/EQUIPMENT/weapon_equip.ini [hunk 13]
```diff
@@ -50907,6 +50907,414 @@ turn_rate = 90
...
+lootable = false
+
+[Gun]
+nickname = gd_ww_gun_laser_medium01
+ids_name = 459784
+;res str
+; Heaven’s Point ($class)
+ids_info = 461329
+;res html
+; \m\b“Heaven’s point” Wardens Weapon\B
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/weapon_equip.ini:51066]: Capital P for point

File: Freelancer/DATA/EQUIPMENT/weapon_equip.ini [hunk 14]
```diff
@@ -50907,6 +50907,414 @@ turn_rate = 90
...
+lootable = false
+
+[Gun]
+nickname = gd_ww_gun_laser_medium02
+ids_name = 459785
+;res str
+; Heaven’s Point MKII ($class)
+ids_info = 461330
+;res html
+; \m\b“Heaven’s point MKII” Wardens Weapon\B
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/weapon_equip.ini:51131]: Point with capital P

File: Freelancer/DATA/EQUIPMENT/weapon_equip.ini [hunk 15]
```diff
@@ -51302,3 +51710,70 @@ auto_turret = false
...
+; Type: $classLong $damageType
+; Hull/Shield Damage: $hullDamage/$energyDamage
+; Range: $range m
+; Projectile Speed: $speed m/s
+; Refire Rate: $refireRate /s
+; Power Draw: $powerUsage
+; Mass: $mass
+; Integrity: $hitpoints
+; \
+; \cYellow\i\bHappy Hollidays\B\I\C
```
Comment (erikroe) [inline Freelancer/DATA/EQUIPMENT/weapon_equip.ini:51749]: Holidays with one L and it is missing the ! at the end like the other equips have it.

**Review by erikroe — CHANGES_REQUESTED** [review]

**Review by erikroe — APPROVED** [review]

**Pull request merged**